src/main/java/ExpressionEvaluator.java

```diff
@@ -0,0 +1,79 @@
+import java.util.*;
+import java.util.function.BiFunction;
+
+public class ExpressionEvaluator {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/manual/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4332639)